rollup.config.js

```diff
@@ -70,18 +70,57 @@ function createCommonJSConfig(input, output) {
   }
 }
 
+function createUMDConfig(input, output) {
+  return {
+    input,
+    output: {
+      file: output,
+      format: 'umd',
+      exports: 'named',
+      name: 'zustand',
+      esModule: true,
+    },
+    external,
+    plugins: [
+      resolve({ extensions }),
+      babelPlugin(getBabelOptions({ ie: 11 })),
+    ],
+  }
+}
+
+function createSystemConfig(input, output) {
+  return {
+    input,
+    output: {
+      file: output,
+      format: 'system',
+      exports: 'named',
+      esModule: true,
+    },
+    external,
+    plugins: [
+      resolve({ extensions }),
+      babelPlugin(getBabelOptions({ ie: 11 })),
```

Comment:
Do systemjs users need old browser support? (they can use umd in such cases?)
otherwise,
```suggestion
      getEsbuild('node12'),
```
would build more efficient output?